repo.yaml

```diff
@@ -0,0 +1,9 @@
+codeowners:
+- owners:
+  - '@joostfaassen'
+  pattern: '*'
+description: 'Webapp Infrastructure as code. An Ansible Collection.'
+license: gpl-v3
+name: ansible-collection-polaris
```

Comment:
yep!

Reply:
Name here cannot be changed as it has to match the GitHub repository name, renaming this would rename the repository.

Reply:
My comment was ambiguous, sorry about that.  I meant "yep" as "this is correct"!